pymc/distributions/distribution.py

```diff
@@ -178,11 +185,55 @@ def _make_rv_and_resize_shape_from_dims(
     return rv_out, resize_shape_from_dims
 
 
+class SymbolicRandomVariable(OpFromGraph):
+    """Symbolic Random Variable
+
+    This is a subclasse of `OpFromGraph` which is used to encapsulate the symbolic
+    random graph of complex distributions which are built on top of pure
+    `RandomVariable`s.
+
+    These graphs may vary structurally based on the inputs (e.g., their dimensionality),
+    and usually require that random inputs have specific shapes for correct outputs
+    (e.g., avoiding broadcasting of random inputs). Due to this, most distributions that
+    return SymbolicRandomVariable create their these graphs at runtime via the
+    classmethod `cls.rv_op`, taking care to clone and resize random inputs, if needed.
+    """
+
+    ndim_supp: int = None
+    """Number of support dimensions as in RandomVariables
+    (0 for scalar, 1 for vector, ...)
+     """
+
+    inline_aeppl: bool = False
```

Comment:
Are there symbolic rvs that have `inline_aeppl` to be `True` by default?

Reply:
Censored does

Reply:
I see, thanks!